jobs/service-fabrik-apiserver/templates/config/bpm.yml.erb

```diff
@@ -1,7 +1,29 @@
 ---
 processes:
 - name: service-fabrik-apiserver
-  executable: /var/vcap/jobs/service-fabrik-apiserver/bin/service-fabrik-apiserver
-  persistent_disk: true
-  limits:
-    open_files: 100000
+  executable: /var/vcap/packages/apiserver/bin/kube-apiserver
+  args:
+  - --anonymous-auth=true
+  - --allow-privileged=false
+  - --apiserver-count=<%= link('service-fabrik-apiserver').instances.size %>
+  - --bind-address=0.0.0.0
+  - --enable-bootstrap-token-auth
+  - --enable-swagger-ui
+  - --etcd-servers=https://<%= p('etcd.url').join(',') %>
+  - --etcd-cafile=/var/vcap/jobs/service-fabrik-apiserver/config/etcd-ca.crt
```

Comment:
Can we use variable from job_properties file path **/var/vcap/jobs/service-fabrik-apiserver/config**

Reply:
There are no jobs_properties file as in bpm I directly run binary and no shell script file is needed. Let me try to declare the variable in the top in this erb.